syntax = "proto3";

package ProtobufTest;

option optimize_for = SPEED;


message AudioData {
	uint32 sender_session = 1;
	uint64 frame_number   = 2;
	bytes opus_data       = 3;
	bool is_terminator    = 16;

	uint32 targetOrContext         = 4;
	repeated float positional_data = 5;
	float volume_adjustment        = 6;
}

message AudioPart {
	uint32 targetOrContext         = 4;
	repeated float positional_data = 5;
	float volume_adjustment        = 6;
}
